base/array.jl

```diff
@@ -791,9 +791,9 @@ function find(testf::Function, A::AbstractArray)
     I
 end
 
-function find(A::AbstractArray)
+function find(A)
     nnzA = countnz(A)
-    I = similar(A, Int, nnzA)
+    I = Vector{Int}(nnzA)
```

Comment:
does this change the dimensionality of the output when the input is an `AbstractArray` ?